templates/header.hbs

```diff
@@ -0,0 +1,16 @@
+    <header>
+        <div class="logo">
+            <img src={{assetSrc robot}} alt="">
+            <p>{{item1_header}}</p>
+        </div>
+
+        <nav>
+            <ul class="nav__links">
+                <li><a href="#">NUESTRO TRABAJO</a></li>
+                <li><a href="#">APRENDAMOS</a></li>
+                <li><a href="#">DOCUMENTOS</a></li>
+                <li><button class="btn">DONACIONES</button></li>
```

Comment:
it should be also a hyperlink with href (<a href="...">DONACIONES</a>), because it needs to redirect to the donations page.

Reply:
Thanks you. Done.